cpp/CMakeLists.txt

```diff
@@ -127,10 +128,10 @@ endif()
 
 if(CMAKE_CXX_COMPILER_ID MATCHES "GNU")
     string(APPEND MEMILIO_CXX_FLAGS_ENABLE_WARNING_ERRORS
-        "-Wno-unknown-warning;-Wno-pragmas;-Wno-deprecated-copy;-Wno-expansion-to-defined;")
+        "-Wno-unknown-warning;-Wno-pragmas;-Wno-deprecated-copy;-Wno-expansion-to-defined;-Wno-stringop-overread;")

```

Comment:
Are we sure we want to suppress this always?

I think this should be fine, the older gcc 9 seems to already accept the flag, and valgrind is surprisingly okay with memilio-test, despite the death tests. We do potentially suppress warnings on faulty string operations, but there are too many false positives to do nothing right now.

Reply:
I agree. The flag is not ideal. Maybe, we should open an issue to keep track of this and solve this better once there are better options.

Reply:
Moved to #1263